components/data-oriented-programming/code/java-bbe/stream-type.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: 'Model data streams'
-description: In data-oriented programming, efficient handling and processing of large amounts of data is vital. Ballerina's built-in stream type enables developers to process data on-demand, apply transformations, filters, and aggregations, and facilitates seamless integration with other data processing operations.
+description: Efficient handling and processing of large amounts of data is vita in data-oriented programming. Ballerina's built-in stream type enables developers to process data on-demand, apply transformations, filters, and aggregations, and facilitates seamless integration with other data processing operations.
```

Comment:
vita typo and I don't see a real reason to change the original text.

Reply:
Original text was updated based on https://github.com/ballerina-platform/ballerina-dev-website/pull/7447#discussion_r1292677644.

Reply:
Same comment. https://github.com/ballerina-platform/ballerina-dev-website/pull/7447#discussion_r1294690780

Reply:
Reverted to the original.